internal/dashboard/business/oceanbase/obtenant.go

```diff
@@ -595,6 +624,31 @@ func PatchTenant(ctx context.Context, nn types.NamespacedName, p *param.PatchTen
 			}
 		}
 	}
+	if alreadyIgnoreDeletion && p.RemoveDeletionProtection {
+		delete(tenant.Annotations, oceanbaseconst.AnnotationsIgnoreDeletion)
+	} else if !alreadyIgnoreDeletion && p.AddDeletionProtection {
+		tenant.Annotations[oceanbaseconst.AnnotationsIgnoreDeletion] = "true"
+	}
+	if len(p.Variables) > 0 {
+		newVars := make([]apitypes.Variable, 0, len(p.Variables))
+		for i := range p.Variables {
+			newVars = append(newVars, apitypes.Variable{
+				Name:  p.Variables[i].Key,
+				Value: p.Variables[i].Value,
+			})
+		}
+		tenant.Spec.Variables = newVars
```

Comment:
overwrite all the parameters and variables?

Reply:
Yes, the same as parameters of obcluster. The front end application will send all parameters here if the user desireds to modify parameters here.